gnuradio-runtime/include/gnuradio/buffer.h

```diff
@@ -135,7 +135,7 @@ namespace gr {
     std::multimap<uint64_t,tag_t>::iterator get_tags_lower_bound(uint64_t x) { return d_item_tags.lower_bound(x); }
     std::multimap<uint64_t,tag_t>::iterator get_tags_upper_bound(uint64_t x) { return d_item_tags.upper_bound(x); }
 
-    // -------------------------------------------------------------------------
+    virtual bool allocate_buffer();
```

Comment:
This was moved from the private section to the public section.  Was this to facilitate overloading?  In that case, it should actually be in the protected section, so derived classes can override it, but it would still not be exposed publicly.  I think.